db/model/Gdoc/GdocHomepage.ts

```diff
@@ -63,8 +63,12 @@ export class GdocHomepage
     _loadSubclassAttachments = async (
         knex: db.KnexReadonlyTransaction
     ): Promise<void> => {
+        const grapherCount = await db.getTotalNumberOfCharts(knex)
+        const nonGrapherExplorerViewCount =
+            await db.getNonGrapherExplorerViewCount(knex)
```

Comment:
Can maybe optimize this a tiny bit by using `Promise.all` to parallelize these two.